### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by DilsherB**
projects added in portfolio

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  font-family: 'Poppins', sans-serif;
+}
+
+.project {
+  background-color: white;
+  padding: 15px;
+  border-radius: 20px;
+  margin: 20px 10px;
+}
+
```
Comment (mckent05) [inline style.css]: According to the project requirements, this should use a `flex` display property. Please fix this 🙏

File: style.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  width: 89px;
+  height: 16px;
+}
+
+#aboutMyself {
+  background-repeat: no-repeat;
+  background-color: white;
+  border-top-right-radius: 50px;
+  padding: 10px 20px 40px 20px;
+}
```
Comment (mckent05) [inline style.css]: According to the project requirements, this section of the page should be aligned using the `flex` display property, please fix this 👍

File: index.html [hunk 3]
```diff
@@ -4,9 +4,139 @@
...
+            <ul class="ulDisplay">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javascrip</li>
+            </ul>
+            <button >See Project</button>
+        </div>
+    </section>
+    <section id="aboutMyself">
+        <h1 id="introHeader">About Myself</h1>
```
Comment (mckent05) [inline index.html]: **OPTIONAL** To improve the SEO of your site, it is not advisable to have more than one `h1` tag on your page. Please fix this and also ensure that your `hx` tags follows a clear hierarchy. 👍

File: index.html [hunk 4]
```diff
@@ -4,9 +4,139 @@
...
+        <button>Get my resume</button>
+        <h2>Languages</h2>
+        <ul>
+            <li class="bgs"> <img src="images/jsLogo.svg" alt="">JavaScript</li>
+            <li class="bgs"> <img src="images/htmlLogo.svg" alt="">HTML</li>
+            <li class="bgs"> <img src="images/cssLogo.svg" alt="">CSS</li>
+        </ul>
+        <h2>Frameworks</h2>
+        <h2>Skills</h2>
+    </section>
```
Comment (mckent05) [inline index.html]: Good job on the job done so far, but there are some differences between the Figma design and your site. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Figma design:
qaaaa

Your design:
axza

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @DilsherB, 

# STATUS: CHANGES REQUIRED ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
Your portfolio is looking good 👍 
All linter checks are passing ✔️ 
You have used the right workflow for this milestone 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DilsherB** [comment]
All the issues of CR are resolved

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @DilsherB , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] Implemented all the changes from the previous reviewer :white_check_mark: 

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DilsherB** [comment]
merging projects branch to main

**Pull request merged**